app/services/back_end_service.rb

```diff
@@ -1,7 +1,7 @@
 class BackEndService
   def self.db_conn
-    Faraday.new('https://spotme-app-api.herokuapp.com')
-    # Faraday.new('http://localhost:4500')
+    # Faraday.new('https://spotme-app-api.herokuapp.com')
+    Faraday.new('http://localhost:4500')
```

Comment:
Make sure to change this back to the heroku url :)

Reply:
Yup!  Once BE PR is merged and I can rerecord cassettes from the FE branch, this will all be passing build pointed to the Heroku URL